src/raiden_libs/matrix.py

```diff
@@ -60,6 +61,10 @@ def message_from_dict(data: Dict[str, Any]) -> Message:
 def deserialize_messages(data: str, peer_address: Address) -> List[SignedMessage]:
     messages: List[SignedMessage] = list()
 
+    if sys.getsizeof(data) >= 1000000:
```

Comment:
This does not check the size of a single message, but the total size of a set of messages. This makes it hard to tell if 1MB is a safe value to check against unless you depend on a specific message batching behavior in the transport..

Reply:
But this might be the attack vector, right? Or should we analyse on a message level?

Reply:
changed it to an per line analysis